Sources/Apodini/Semantic Model Builder/OpenAPI/OpenAPIInterfaceExporter.swift

```diff
@@ -3,44 +3,83 @@
 //
 
 @_implementationOnly import OpenAPIKit
-@_implementationOnly import struct Vapor.Abort
+@_implementationOnly import Vapor
 import Foundation
 
 class OpenAPIInterfaceExporter: StaticInterfaceExporter {
     static var parameterNamespace: [ParameterNamespace] = .individual
 
     let app: Application
     var documentBuilder: OpenAPIDocumentBuilder
-    let configuration: OpenAPIConfiguration
+    var configuration: OpenAPIConfiguration
```

Comment:
As far as I can see this could be made a constant

Reply:
👍 09f5109